src/lib.rs

```diff
@@ -74,6 +74,7 @@ mod interface;
 
 pub mod epd1in54;
 pub mod epd1in54b;
+pub mod epd2in13_v2;
```

Comment:
Which version of the epd is it exactly? Better go for epd2in13bc or epd2in13d depending on which one it is exactly or if it is the epd2in13a in v2 then leave it like it is. I am just a bit confused by the red buffer you talk about somewhere above

Reply:
It's the «regular» one, not B, C nor D. They are only selling v2 now.
The RED buffer is the name used by the controller, so I've kept the name to be consistent with it. They are using the RED ram for the partial refresh.